runtime-modules/content/src/benchmarks/benchmarking.rs

```diff
@@ -2249,147 +2261,1096 @@ benchmarks! {
+    //   - max channel assets
+    //   - max collaborators
+    // - video has max size
+    //   - max video assets
+    // - nft limits are set
+    // - nft owner is channel owner
+    // INPUT COMPLEXITY
+    cancel_open_auction_bid {
+        let (
+            video_id,
+            (curator_account_id, actor, channel_id, _)
+        ) = setup_worst_case_scenario_mutable_video::<T>(
+            Some(T::MaxNumberOfAssetsPerVideo::get()),
+            T::StorageBucketsPerBagValueConstraint::get().max() as u32,
+        )?;
+
+        let (owner_account, owner_id) = member_funded_account::<T>(DEFAULT_MEMBER_ID);
+        let nft_owner_actor = ContentActor::<T::CuratorGroupId, T::CuratorId, T::MemberId>::Member(owner_id);
+
+        let (_, participant_id, participant_account_id) = setup_nft_in_open_auction::<T>(
+            curator_account_id.clone(),
+            actor,
+            video_id,
+            false,
+        )?;
+
+        set_all_channel_paused_features::<T>(channel_id);
+        let origin = RawOrigin::Signed(participant_account_id.clone());
+
+        fastforward_by_blocks::<T>(2u32.into());
+
+        let bid = add_open_auction_bid::<T>(participant_account_id, participant_id, video_id);
+        fastforward_by_blocks::<T>(10u32.into()); // skip bid lock
+
+    }: _(origin, participant_id, video_id)
+        verify {
+            assert_eq!(Pallet::<T>::open_auction_bid_by_video_and_member(video_id, participant_id).amount, 0u32.into());
+        }
+
+    // WORST CASE SCENARIO
+    // STATE COMPLEXITY
+    // - curator owned channel
+    // - curator number is max
+    // - curator has max number of agent permissions
+    // - channel has max size:
+    //   - all feature paused (except necessary ones for extr to succeed)
+    //   - max channel assets
+    //   - max collaborators
+    // - video has max size
+    //   - max video assets
+    // - nft limits are set
+    // - complete payment has max complexity:
+    //   - nft owner is a member (different from channel owner)
+    //   - royalty is non-zero
+    //   - `price - royalty` is non-zero
+    // INPUT COMPLEXITY
+    pick_open_auction_winner {
+        let (
+            video_id,
+            (curator_account_id, actor, channel_id, _)
+        ) = setup_worst_case_scenario_mutable_video::<T>(
+            Some(T::MaxNumberOfAssetsPerVideo::get()),
+            T::StorageBucketsPerBagValueConstraint::get().max() as u32,
+        ).unwrap();
+
+        let ((nft_owner_actor, owner_account), participant_id, participant_account_id) = setup_nft_in_open_auction::<T>(
+            curator_account_id.clone(),
+            actor,
+            video_id,
+            true,
+        ).unwrap();
+
+        set_all_channel_paused_features::<T>(channel_id);
+        let origin = RawOrigin::Signed(owner_account.clone());
+
+        fastforward_by_blocks::<T>(2u32.into());
+
+        let bid = add_open_auction_bid::<T>(participant_account_id, participant_id, video_id);
```

Comment:
Same issue as with `settle_english_auction` - the bid is `1`, so worst case scenario is not tiggered

Reply:
I set that value to nft_buy_now_price - 1.
The min_starting_price is 10 hovewer...